modules/libevdev/1.13.2/patches/add_build_file.patch

```diff
@@ -0,0 +1,137 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,123 @@
++load("@rules_python//python:defs.bzl", "py_binary")
++
++# This is the public target to depend on.
++# Headers are C++ compatible.
++cc_library(
++    name = "evdev",
```

Comment:
people like to add an alias to this target (so `alias(name="libevdev", actual=":evdev")`) so that they can simply write `cc_library(..., deps = ["@libevdev"])`

Reply:
Done.